src/dialogs/quick-open/ha-quick-open-dialog.ts

```diff
@@ -0,0 +1,249 @@
+      this._commandItems,
+      this._itemFilter
+    );
+
+    return html`
+      ${items.map(
+        ({ text, domain, service, serviceData, onClick }) => html`
+          <mwc-list-item
+            @click=${onClick}
+            .domain=${domain}
+            .service=${service}
+            .serviceData=${serviceData}
+            graphic="icon"
+          >
+            <ha-icon .icon=${domainIcon(domain)} slot="graphic"></ha-icon>
+            ${text}
+          </mwc-list-item>
+        `
+      )}
+    `;
+  }
+
+  protected renderEntityList() {
+    const entities = this._filterEntityItems(
+      Object.keys(this.hass.states),
+      this._itemFilter
+    );
+
+    return html`
+      ${entities.map((entity_id) => {
+        const domain = computeDomain(entity_id);
+        return html`
+          <mwc-list-item
+            @click=${this._entityMoreInfo}
+            .domain=${domain}
+            graphic="icon"
+          >
+            <ha-icon .icon=${domainIcon(domain)} slot="graphic"></ha-icon>
+            ${entity_id}
+          </mwc-list-item>
+        `;
+      })}
+    `;
+  }
+
+  private _entityFilterChanged(ev: PolymerChangedEvent<string>) {
+    const newFilter = ev.detail.value;
+
+    if (newFilter.startsWith(">")) {
+      this._commandMode = true;
+      this._itemFilter = newFilter.substring(1);
+    } else {
+      this._commandMode = false;
+      this._itemFilter = newFilter;
+    }
+  }
+
+  private _generateCommandItems(): CommandItem[] {
+    const reloadableDomains = componentsWithService(this.hass, "reload").sort();
+
+    return reloadableDomains.map((domain) => ({
+      text:
+        this.localize(`ui.panel.quick-open.commands.reload.${domain}`) ||
+        this.localize(
+          "ui.panel.quick-open.commands.reload.reload",
+          "domain",
+          domainToName(this.localize, domain)
+        ),
+      domain,
+      service: "reload",
+      onClick: () => this._executeCommand,
+    }));
+  }
+
+  private _filterCommandItems(
+    items: CommandItem[],
+    filter: string
+  ): CommandItem[] {
+    return items
+      .filter(({ text }) =>
+        fuzzySequentialMatch(filter.toLowerCase(), text.toLowerCase())
+      )
+      .sort((itemA, itemB) => compare(itemA.text, itemB.text));
+  }
+
+  private _filterEntityItems(
+    entity_ids: HassEntity["entity_id"][],
+    filter: string
+  ): HassEntity["entity_id"][] {
+    return entity_ids
+      .filter((entity_id) =>
+        fuzzySequentialMatch(filter.toLowerCase(), entity_id)
+      )
+      .sort();
+  }
+
+  private async _executeCommand(ev: Event) {
+    const eventData = {
+      domain: (ev.target as any).domain,
+      service: (ev.target as any).service,
+      serviceData: (ev.target as any).serviceData,
+    };
+
+    await this.hass.callService(
+      eventData.domain,
+      eventData.service,
+      eventData.serviceData
+    );
+
+    fireEvent(this, "hass-service-called", eventData);
```

Comment:
```suggestion
```

Reply:
Oh whoops. This (and several of your other suggestions) weren't pushed last night.

Reply:
Done